pkgs/development/python-modules/asttokens/1.6.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, fetchPypi, buildPythonPackage, pythonOlder, isPyPy
```

Comment:
Wrong file

Reply:
Repaired. Apologies!

Reply:
Why are you adding a file called 1.6.nix in asttokens that contains a definition for a package called asteroid? Does Thonny depend on asteroid?

Reply:
No, this was a commit error.